var/spack/repos/builtin/packages/rocm-tensile/package.py

```diff
@@ -14,48 +14,54 @@ class RocmTensile(CMakePackage):
     url      = "https://github.com/ROCmSoftwarePlatform/Tensile/archive/rocm-3.8.0.tar.gz"
     maintainers = ['srekolam', 'arjun-raj-kuppala']
 
+    version('4.0.0', sha256='cf105ce8c3e352d19713b3bf8bda77f25c1a692c4f2ca82d631ba15523ecc1cd')
+    version('3.10.0', sha256='8d5b50aadfa56a9195e4c387b8eb351c9b9b7671b136b624e07fe28db24bd330')
+    version('3.9.0', sha256='17a011f8c3433d4f8c2dddabd5854cf96c406d24592b3942deb51672c570882e')
     version('3.8.0', sha256='c78a11db85fdf54bfd26533ee6fa98f6a6e789fa423537993061497ac5f22ed6')
     version('3.7.0', sha256='488a7f76ea42a7601d0557f53068ec4832a2c7c06bb1b511470a4e35599a5a4d')
     version('3.5.0', sha256='71eb3eed6625b08a4cedb539dd9b596e3d4cc82a1a8063d37d94c0765b6f8257')
 
-    amdgpu_targets = ('all', 'gfx803', 'gfx900', 'gfx906', 'gfx908')
+    tensile_architecture = ('all', 'gfx803', 'gfx900', 'gfx906', 'gfx908')
 
-    variant('amdgpu_target', default='all', multi=True, values=amdgpu_targets)
+    variant('tensile_architecture', default='all', values=tensile_architecture, multi=False)
 
     depends_on('cmake@3:', type='build')
+    depends_on('boost@1.58.0', type=('build', 'run'), when='@3.9.0:')
```

Comment:
It's already listed a few lines below
```suggestion

```
also I don't understand `type=run` for boost.

Reply:
Let me check this..

Reply:
Can you still remove this line? Boost is added as dependency on line 42